<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="utf-8"/><meta content="width=device-width,initial-scale=1" name="viewport"/><title>Engineering | iGEM Stockholm</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="https://2020.igem.org/Template:Stockholm/css/contentCSS?action=raw&amp;ctype=text/css" rel="stylesheet"/></head><body><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://2020.igem.org/Team:Stockholm"></a><button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><i class="fas fa-bars"></i></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown" role="button">Project</a><div aria-labelledby="navbarProjectDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Description">Description</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Design">Design</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Model">Model</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Engineering">Engineering</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Experiments">Experiments</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Notebook">Notebook</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Results">Results</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Contribution">Contribution</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Implementation">Implementation</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarPartsDropdown" role="button">Parts</a><div aria-labelledby="navbarPartsDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Parts">Parts</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Primers">Primers</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarHuman PracticesDropdown" role="button">Human Practices</a><div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Education">Education</a></div></li><li class="nav-item"><a class="nav-link" href="https://2020.igem.org/Team:Stockholm/Safety">Safety</a></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarAboutDropdown" role="button">About</a><div aria-labelledby="navbarAboutDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Team">Team</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Attributions">Attributions</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Sponsors">Sponsors</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarAwardsDropdown" role="button">Awards</a><div aria-labelledby="navbarAwardsDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Sustainable">Sustainable</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Medal_Criteria">Medal Criteria</a></div></li></ul></div><div class="d-flex" id="themeSwitchWrapper"><i class="far fa-sun"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input id="themeSwitchInput" type="checkbox"/><span class="slider round"></span></label></div><i class="far fa-moon"></i></div></div></nav><div class="parent"><header class="d-flex justify-content-center align-items-center"><div class="container"><h1>Engineering</h1><p class="lead pl-1">A sample page for the theme.</p><a class="arrow" href="#wave"><i class="fas fa-chevron-down"></i></a><img class="topimg" src="https://2020.igem.org/wiki/images/6/63/T--Stockholm--img--Contents--topimg_engineer.png"/><div class="wave wave1" id="wave"></div><div class="wave wave2"></div><div class="wave wave3"></div></div></header><main><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Overview</h1><p>iGEM Stockholm 2020 presents its engineering cycle (Figure 1).</p><div class="image"><img alt="Engineering cycle" src="https://2020.igem.org/wiki/images/5/59/T--Stockholm--img--Engineering--cycle.png" style="width: 100%"/><p>Figure 1: Engineering cycle</p></div><p>In order to build a scientifically rigorous project, iGEM Stockholm proceeded by following the engineering cycle and improving little by little each and every experiment performed during the summer. We started by finding a problem we wanted to solve and gathering information about it until a clear goal could be defined. Then we divided our project into subtasks and modules. We designed experiments that would help us build and prove the viability and success of our Microbial Fuel Cell (MFC) biosensor. We modelled our main goal, the oscillatory circuit that would help detect and identify different pollutants, in-silico. We built a MFC and assembled and tested our constructs in parallel. We characterized the MFC and the constructs, analysed the results and drew conclusions for each. For each problem encountered, we followed the cycle again to optimize our experiments and get better results.</p><h1>1. Problem identification</h1><p>In the first few months of iGEM, we gathered to find a project that was interesting and meaningful to the full team. In April-June, it became clear that we wanted a project related to the pollution of the Baltic sea, as this was a problem which we had all heard of and that directly affects the area where we live.</p><h1>2. Gathering information</h1><p>We dived into the problem of sea pollution by reading literature and contacting researchers that worked both in the field of microbiology and pollution. We looked for reports on pollution in the Baltic sea to see what pollutants were the most problematic and what their origin was. In particular, the Helsinki Commission published in the spring a series of reports [1],[2],[3],[4] highlighting the levels and effects of dioxins, polychlorinated biphenyls (PCBs), flame retardants such as polybrominated diphenyl ethers and polybrominated biphenyls, diclofenac, perfluorooctanesulfonic acid (PFOS) and polyfluoroalkyl substances (PFASs). We chose to focus on two types of pollutants PCBs and PFOA, for which we found promoter parts engineered and/or studied by previous iGEM teams (BBa_K2911000 and BBa_K1155001).</p><p>Once we chose to work on a microbial fuel cell for these pollutants’ detection, we figured that two approaches could be taken: either increasing the electrical production of the cell or taking a novel approach to distinguish and quantify pollutants. Because previous approaches to increase electricity production had not been very successful, we decided to use synthetic biology to create unique electrical signals for different pollutants.</p><p><em>Shewanella oneidensis MR-1</em> is commonly used in microbial fuel cells for electricity production thanks to its mtrb gene. However, most iGEM teams create parts that work in <em>E. coli</em> and we wanted to contribute to the creation of new parts that could be used by as many teams as possible in the future. Besides, most microbiology techniques and protocols are easily adapted and troubleshooted in <em>E. coli</em>. Thus, we developed the idea of having a modular system: a system where most of the synthetic biology work of pollutant detection would be performed on <em>E. coli</em>, and where the electrical output would be given by <em>Shewanella oneidensis MR-1</em>. To connect these two modules, we chose to take advantage of quorum sensing molecules, molecules that can transmit signals between different cells.</p><h1>3. Defining a goal</h1><p>The final goal of our project was to engineer <em>E. coli</em> and <em>S. oneidensis</em> strains so that their growth in MFC, in the presence of pollutants could give an oscillating electrical output that would be unique for each pollutant.</p><h1>4. Plan and define subtasks</h1><p>Since our project is by essence modular, it was easy to divide the tasks: we had to work in parallel on</p><ul><li>engineering E. coli so that it detects pollutants and emits quorum sensing (QS) molecules as a result;</li><li>engineering a Shewanella oneidensis MtrB knockout strain (unable to produce electricity in normal conditions) so that the received QS signal from E. coli activates a conditional electrical pathway (production of electricity only if a pollutant is detected);</li><li>designing a genetic circuit that gives a unique oscillating output when a certain pollutant is detected;</li><li>building a functional microbial fuel cell in which the strains could perform their activities.</li></ul><div class="image"><img alt="Engineering division" src="https://2020.igem.org/wiki/images/9/94/T--Stockholm--img--Engineering--division.png" style="width: 100%"/><p>Figure 2: Engineering division</p></div><h1>5. Design</h1><h2>A. Circuit</h2><p>We used the SnapGene software to <a href="https://2020.igem.org/Team:Stockholm/Design">design</a> the following <a href="https://2020.igem.org/Team:Stockholm/Parts">parts</a> and circuits using the Biobrick format.</p><table><caption id="table1listofconstructs">Table 1: List of constructs.</caption><thead><tr><th>Biobrick</th><th>Code</th><th>Construct</th><th>Aim</th></tr></thead><tbody><tr><td>BBa_K3440000</td><td>A</td><td>Pconst-RBS-LuxI-Myc-DT</td><td>Show that LuxI can be expressed under constitutive promoter</td></tr><tr><td>BBa_K3440001</td><td>B</td><td>PbphR1-RBS-mCherry-DT</td><td>Test bphR1 promoter leakiness and compare expression with/without PCB.</td></tr><tr><td>BBa_K3440002</td><td>C</td><td>Pprma-RBS-mCherry-DT</td><td>Test leakiness of Pprma</td></tr><tr><td>BBa_K3440003</td><td>D</td><td>Plux-RBS-GFP</td><td>Test leakiness of Plux</td></tr><tr><td>BBa_K3440004</td><td>E</td><td>Prhl-RBS-GFP</td><td>Test leakiness of Prhl</td></tr><tr><td>BBa_K3440005</td><td>F</td><td>Pprma-RBS-LuxI-Myc</td><td>Show that LuxI can be expressed under prma promoter, and compare expression with/without PFOA.Final construct <em>E. coli</em> PFOA.</td></tr><tr><td>BBa_K3440006</td><td>G</td><td>Pconst-RBS-RhlI-Myc</td><td>Show that RhlI can be expressed under constitutive promoter</td></tr><tr><td>BBa_K3440007</td><td>I</td><td>Pconst-RBS-AiiA-Myc</td><td>Show that AiiA can be expressed under constitutive promoter</td></tr><tr><td>BBa_K3440008</td><td>J</td><td>Pconst-RBS-RhlR-Myc-DT</td><td>Show that RhlR can be expressed under Pconst</td></tr><tr><td>BBa_K3440009</td><td>L</td><td>Pconst-RBS-bphR2-Myc-DT</td><td>Show that bphR2 can be expressed under Pconst</td></tr><tr><td>BBa_K3440010</td><td>M</td><td>Pbphr1-RBS-LuxI-Myc</td><td>Show that LuxI can be expressed under bphR1 promoter, and compare expression with/without PCB.</td></tr><tr><td>BBa_K3440011</td><td>N</td><td>Pconst-RBS-GFP-DT</td><td>Show that the constitutive promoter works</td></tr><tr><td>BBa_K3440012</td><td>O</td><td>Pconst-RBS-RHlR-Myc-DT-Prhl-RBS-GFP</td><td>Show that Prhl promoter can be activated in the presence of AHL and RhlR expression</td></tr><tr><td>BBa_K3440013</td><td>P</td><td>Pconst-RBS-LuxR-Myc</td><td>Show that LuxR can be expressed under Pconst</td></tr><tr><td>BBa_K3440014</td><td>Q</td><td>Pconst-RBS-LuxR-Myc-DT-Plux-RBS-GFP</td><td>Show that Plux promoter can be activated in the presence of AHL and LuxR expression</td></tr><tr><td>BBa_K3440015</td><td>K</td><td>RhlI-RBS-Aiia</td><td>Intermediate part used for construction</td></tr><tr><td>BBa_K3440016</td><td>I/Ch5</td><td>Pconst-RBS-Mtrb-Myc</td><td>Show that Mtrb can be expressed under constitutive promoter</td></tr><tr><td>BBa_K3440017</td><td>L3B</td><td>Pconst-RBS-bphR2-Myc-DT-Pbphr1-RBS-mCherry</td><td>Show that bphR1 promoter can be activated by expression of bphR2 (and better with PCB)</td></tr><tr><td>BBa_K3440018</td><td>P32</td><td>Pconst-RBS-bphR2-Myc-DT-Pbphr1-RBS-LuxI</td><td>Full construct for detection of PCB and expression of LuxI. Final construct PCB in <em>E. coli</em></td></tr><tr><td>BBa_K3440019</td><td>Osc1</td><td>Pconst-RBS-LuxR-DT-Plux-RBS-mtrb-DT</td><td>Show that mtrb can be expressed under Plux in the presence of AHL.</td></tr><tr><td>BBa_K3440020</td><td>Osc2</td><td>Pconst-RBS-RhlR-DT-Prhl-RBS-RhlI-RBS-AiiA-RBS-GFP-DT</td><td>GFP reporter of oscillation in E.coli</td></tr><tr><td>BBa_K3440021</td><td>Osc3</td><td>Plux-RBS-Mtrb-DT-Pconst-RBS-RhlR-DT-Prhl-RBS-RhlI-RBS-AiiA-RBS-LuxR-DT</td><td>Electricity production under oscillatory circuit in <em>Shewanella</em>.</td></tr></tbody></table><p>Because many of these constructs were long and would each require many days of assembling biobricks, we decided to order most of them from IDT Technologies as gene blocks with convenient restriction sites in optimal positions and tags to allow characterisation. We used those gene blocks as a starting point for our experiments.</p><p>The full circuit we designed for detection of PCB (it is similar for PFOA) is the following (Figure X): the first construct produces bphR2 constitutively. bphR2 can form a complex with the PCB pollutant, which can activate the bphR1 promoter. LuxI is placed under this promoter and produces a QS molecule. The fourth construct produces RhlR constitutively, and RhlR can bind to C4-HSL to induce Prhl promoter in the third construct, which induces expression of LuxR, RhlI and AiiA. RhlI produces C4-HSL, which creates a positive feedback loop. On the contrary, AiiA degrades C4-HSL, creating the oscillations. LuxR binds to the 3OC6-HSL produced upon detection of PCB by the first construct. The complex can activate Plux promoter, which controls the expression of mtrb, the gene responsible for electricity production in <em>Shewanella</em>.</p><div class="image"><img alt="PCB Circuit" src="https://2020.igem.org/wiki/images/3/3e/T--Stockholm--img--Engineering--PCB_circuit.png" style="width: 100%"/><p>Figure 3: PCB Circuit</p></div><h2>B. MFC</h2><p>The MFC set-up was designed thanks to literature and discussions with Amirreza Khataee from the Applied Electrochemistry department at KTH. We decided to go for a liquid-air hybrid set-up, with one reservoir filled with media and another filled with air. Circulation between the MFC and the reservoirs is created by a pump at 0.1rpm. The chosen electrodes were each made of 2 squares of graphite papers stacked together. The cation exchange membrane used was a NafionTM. The disassembled microbial fuel cell is shown in Figure X. The circuit was assembled as shown in Figure X.</p><div class="image"><img alt="Disassembled MFC" src="https://2020.igem.org/wiki/images/4/46/T--Stockholm--img--Engineering--mfc_disassembled.jpg" style="width: 100%"/><p>Figure 4: Disassembled MFC</p></div><h1>6. Model</h1><p>The circuits were designed with the aim of creating a unique oscillatory electrical output when a certain pollutant is detected by <em>E. coli</em>. To prove that the constructs could give an oscillatory circuit, mathematical simulations were performed. The obtained <a href="https://2020.igem.org/Team:Stockholm/Model">models</a> confirmed that the design was theoretically capable of giving oscillations when pollutants are present.</p><h1>7. Build and experiment</h1><h2>A. Parts</h2><p>For each G-block, we performed (with regards to each particular construct’s requirements) the following series of <a href="https://2020.igem.org/Team:Stockholm/Experiments">experiments</a> and characterisations (Figure X). We first resuspended the G-blocks, PCR amplified them, and then proceeded to insert them in the pSB1C3 plasmid via restriction digestion with EcoRI and PstI enzymes. We ran gels to check the sizes of restriction digested parts, then ligated overnight with pSB1C3. All of this was possible because the G-blocks were designed with predetermined, specific restriction sites. We then transformed <em>E. coli</em> TOP 10 cells with the plasmids. We picked colonies on plates, ran PCR amplification and gels to check if the sizes were as expected and sent the potentially correct ones for sequencing at Microsynth AG. The selected colonies were used to make glycerol stocks and plasmid preparations. OD600 measurements were made for all, and then characterization experiments were performed. For parts containing GFP or mCherry genes, fluorescence measurements were made with a plate reader. For those with a Myc-tag, Western blots were made. We used <em>Chromobacterium violaceum</em> as a marker of AHL production (colonies turning violet when AHLs are detected). Finally, expression was compared between situations with and without pollutants (after viability tests in the presence of pollutants).</p><div class="image"><img alt="Gblock processing" src="https://2020.igem.org/wiki/images/5/52/T--Stockholm--img--Engineering--gblock_processing.png" style="width: 100%"/><p>Figure 5: Gblock processing</p></div><h2>B. MFC</h2><p>We determined from literature on <em>Shewanella oneidensis</em>’ metabolism that <em>S. oneidensis</em> could grow, using lactate as a carbon source to optimise electricity production ([5]). As a result, we chose lactate as an additive to our minimal M9 medium. We decided to check that <em>Shewanella oneidensis</em> MR1 could be used in our MFC set-up to give an electrical output thanks to its mtrb gene. In order to do so, we had to prove that a biofilm could grow on the anode and that this biofilm did not grow when the mtrb gene was knocked out. Thus, we acquired two strains of <em>S. oneidensis</em> MR1; a wild type from Ute Römling’s lab at Karolinska Institutet, Stockholm, Sweden, and a mtrb knockout strain from Johannes Gescher, Karlsruhe Institute of Technology, Germany. We decided to monitor <em>S. oneidensis</em> biofilm growth by measuring the open cell voltage of the circuit with an imposed resistance of 1kΩ. Thanks to Ohm’s law U=RI, the current could then be determined.</p><p>Our first experiment consisted in putting the wild type strain on the electrodes, then assembling the MFC with LB broth in a reservoir, and finally assembling the final circuit (Figure X) and measuring the voltage continuously. After a few days, we changed the media for fresh LB and continued to monitor the voltage. We then changed the media for minimal M9 media with lactate as a carbon source and tried to check that the biofilm growth was enhanced after a period of adaptation to the new media. Our second experiment was a control to check the necessity of having the mtrb gene for electricity production. We assembled the circuit with the mtrb knockout on the electrodes this time and measured current with LB as media.</p><div class="image"><img alt="MFC Setup. 1/green: pump, 2/yellow: air, 3/purple: media, 4/red: resistor, 5/blue: MFC" src="https://2020.igem.org/wiki/images/7/7b/T--Stockholm--img--Engineering--mfc.png" style="width: 100%"/><p>Figure 6: MFC Setup. 1/green: pump, 2/yellow: air, 3/purple: media, 4/red: resistor, 5/blue: MFC</p></div><h1>8. Analyse the results</h1><p>Most of our constructs were successfully designed and transformed into <em>E. coli</em> TOP 10 cells. The sequencing results confirmed what our gels predicted for fifteen of our colonies. The specific results for each experiment (viability, transformation, Western blots, fluorescence) are presented in Results. We decided to focus on the MFC engineering <a href="https://2020.igem.org/Team:Stockholm/Results">Results</a> here.</p><p>First, we observed with the open cell voltage over time curves that a biofilm could grow when <em>S.oneidensis</em>, wild-type was inoculated in the MFC and fed with an LB media (Figure X). The voltage increased slowly and then underwent a high increase when the LB media was renewed. Seeing that the biofilm had grown, we changed the media to a minimal M9 media with regular lactate addition. As expected, the biofilm managed to adapt to this new media. At first, the voltage dropped and then it slowly grew back as it adapted. When we disassembled the MFC, we observed the biofilm directly on the membrane (Figure X).</p><p>Those results were compared to the curves for the mtrb knockout strain (Figure X). The Figure shows that the mtrb knockout could not produce a biofilm because it was unable to produce electricity, as predicted by literature. This result was confirmed when we disassembled the MFC and the membrane was free from any biofilm (Figure X).</p><div class="image"><img alt="WT" src="https://2020.igem.org/wiki/images/0/03/T--Stockholm--img--Engineering--Shewanella_Oneidensis_MR-1_Wildtype.png" style="width: 100%"/><p>Figure 7: WT</p></div><div class="image"><img alt="Knockout" src="https://2020.igem.org/wiki/images/9/94/T--Stockholm--img--Engineering--Shewanella_Oneidensis_MR-1_MtrB_Knockout.png" style="width: 100%"/><p>Figure 8: Knockout</p></div><div class="image"><img alt="A: no biofilm B: with biofilm" src="https://2020.igem.org/wiki/images/3/35/T--Stockholm--img--Engineering--biofilm.png" style="width: 100%"/><p>Figure 9: A: no biofilm B: with biofilm</p></div><h1>9. Discuss and conclude</h1><p><em><strong>An example of engineering success: our MFC success story</strong></em> We managed to create and assemble a fully functional microbial fuel cell in which we have proved that a <em>Shewanella oneidensis</em> strain can form a biofilm on the anode when it is equipped with the mtrb gene. We have optimised electricity production by cultivating this biofilm with a minimal media to which we regularly added lactate to favour the electrical pathway. Finally, we have proved that this biofilm cannot grow without the gene. Thus, our biosensor cannot give a signal without pollutants, but will be able to give an electrical output when pollutants are detected and a construct expressing in an oscillatory manner the mtrb gene can be induced when pollutants are present thanks to the design of appropriate synthetic circuits.</p><h1>10. Identify a problem to solve at i+1 = troubleshooting</h1><p>During the summer, we had to optimise all types of experiments, from restriction digestions to Western blots. The key to troubleshooting is to find the origin of the problem. Most often, using positive and negative controls can both help you prove that your results are scientifically correct and help you find where problems occurred in an experiment. However, you might still encounter problems even with proper controls. We faced many problems with Western Blots, so we decided to share with you our very own troubleshooting checklist for Western Blots. <strong>Quick troubleshooting list to help you when a protein does not appear on a Western blot:</strong></p><ol><li><p>Optimize your Western Blot protocol</p><ol><li>Make sure that you have appropriate positive and negative controls</li><li>Optimize your cell lysis</li><li>Think that your intensity might be too low if you have several products on the same blot, and one is overexpressed compared to the others. In that case you can either have separate gels or stop exposing the overexpressed product.</li><li>If your protein might be expressed in another growth phase, adjust the timing.</li><li>In our case, some proteins were expressed in the presence of AHLs and contaminants (under inducible promoters). If such is the case for you, try to increase the levels of inducers.</li></ol></li><li><p>For cells that express GFP or mCherry or any colored/fluorescent protein, you might want to quickly check to the naked eye with a microscope under the right wavelength that expression occurs. Fluorescence measurements should also be performed and calibrated based on OD600.</p></li><li><p>Check your sequence. Sequencing your part is necessary, and you might as well do it before starting your Western Blots. The reason for this is that even if you have a band at the right size, it might still be another protein due to a substitution. Finally, if your western blot does not work, you can use sequencing to see if the problem is at the DNA level (a mutation to a stop codon could be an explanation).</p></li><li><p>Check your mRNA. If your DNA sequence is correct and you still don’t have results on your blot, then you can try to see if the problem is at the RNA level by performing an RNA extraction.</p></li><li><p>Run a background check on your proteins to understand how they are expressed in their original host, and if possible find a paper in which it was expressed in your host. In particular, check the location of the product and the level of expression obtained in literature.</p></li><li><p>Check your cells. It might be that the protein product is toxic for the cell or that E. coli does not prefer its presence. The product might be easily degraded by protease. Thus, you might want to test growth with and without the construct and observe the number of colonies to see how your host reacts to the product. If that is the problem, then you need to change your host.</p></li><li><p>Coomassie staining can also help you to check your cells by counting total protein expression.</p></li><li><p>To increase expression of the protein, you can clone your plasmid on a low copy number plasmid or under an inducible promoter.</p></li></ol></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Allen, M. J., &amp; Sheridan, S. C. (2015).</p><cite>Mortality risks during extreme temperature events (ETEs) using a distributed lag non-linear model.</cite><p><span class="journalTitle">International Journal of Biometeorology</span> <span class="journalInfo">62(1), 57-67.</span></p><a class="in-text" href="https://doi.org/10.1007/s00484-015-1117-4" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Mortality risks during extreme temperature events (ETEs) using a distributed lag non-linear model." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Rosano, A., Bella, A., Gesualdo, F., Acampora, A., Pezzotti, P., Marchetti, S., ... &amp; Rizzo, C. (2019).</p><cite>Investigating the impact of influenza on excess mortality in all ages in Italy during recent seasons (2013/14-2016/17 seasons).</cite><p><span class="journalTitle">International Journal of Infectious Diseases</span> <span class="journalInfo">88, 127-134.</span></p><a class="in-text" href="https://doi.org/10.1016/j.ijid.2019.08.003" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Investigating the impact of influenza on excess mortality in all ages in Italy during recent seasons (2013/14-2016/17 seasons)." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Ingalls, B. P. (2013).</p><cite>Mathematical modeling in systems biology: An introduction.</cite><p><span class="details">MIT Press.</span></p><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author"></p><cite>Agriculture: Crop production: Sugarcane. TNAU Agritech Portal.</cite><p><span class="details">(March 15, 2019). Retrieved on June 22, 2020. from </span><a class="in-text" href="https://google.com" rel="noopener" target="_blank">https://google.com</a></p><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Author Name. (n.d.).</p><cite>Agriculture: Crop production: Sugarcane. TNAU Agritech Portal.</cite><p><span class="details">Retrieved on June 22, 2020. from </span><a class="in-text" href="https://google.com" rel="noopener" target="_blank">https://google.com</a></p><a class="in-text" href="#intext5">Back to text</a></li></ol></article></div></div></div></main><footer><div class="container"><div class="social_media"><p>Follow iGEM Stockholm on social media</p><a href="https://www.instagram.com/igemstockholm/"><i class="fab fa-instagram"></i></a><a href="https://www.facebook.com/igemstockholm"><i class="fab fa-facebook-f"></i></a><a href="https://www.youtube.com/channel/UCh_a6JvWdh6N_i5tYFFcpyw"><i class="fab fa-youtube"></i></a><a href="https://twitter.com/igemstockholm"><i class="fab fa-twitter"></i></a><a href="https://www.linkedin.com/company/igemstockholm"><i class="fab fa-linkedin-in"></i></a><a href="https://igem.se/"><i class="fas fa-globe-europe"></i></a><a href="mailto:info@igem.se"><i class="far fa-envelope"></i></a></div><hr/><div class="sponsors"><img class="kth" src="https://2020.igem.org/wiki/images/9/94/T--Stockholm--img--Sponsors--kth.png"/><img class="ki_innovations" src="https://2020.igem.org/wiki/images/f/f1/T--Stockholm--img--Sponsors--ki_innovations.png"/><img class="mf" src="https://2020.igem.org/wiki/images/a/a0/T--Stockholm--img--Sponsors--mf.png"/><img class="snapgene" src="https://2020.igem.org/wiki/images/f/f2/T--Stockholm--img--Sponsors--snapgene.png"/><img class="neb" src="https://2020.igem.org/wiki/images/9/9c/T--Stockholm--img--Sponsors--neb.png"/><img class="idt" src="https://2020.igem.org/wiki/images/4/45/T--Stockholm--img--Sponsors--idt.png"/><img class="redbull" src="https://2020.igem.org/wiki/images/0/03/T--Stockholm--img--Sponsors--redbull.png"/><img class="benchling" src="https://2020.igem.org/wiki/images/9/98/T--Stockholm--img--Sponsors--benchling.png"/></div><hr/><div class="credits"><p>Sample template built using the iGEM Wiki Starter Pack by BITS Goa.</p><p>Code released under the MIT license.</p><p>Based on <a href="https://getbootstrap.com">Bootstrap</a> and themes <a href="https://bootswatch.com/flatly/">Flatly</a> and <a href="https://bootswatch.com/darkly/">Darkly</a> from <a href="https://bootswatch.com/">Bootswatch</a>.</p><p>Some content from the <a href="https://2020.igem.org/Team:Example">iGEM Example Wiki</a>. Images from <a href="https://unsplash.com">Unsplash</a>. Web fonts from <a href="https://fonts.google.com">Google</a>.</p></div></div></footer></div><script src="https://2020.igem.org/Template:Stockholm/content-bundleJS?action=raw&amp;ctype=text/javascript"></script></body></html>